ipn/prefs.go

```diff
@@ -195,6 +195,9 @@ type Prefs struct {
 	// and CLI.
 	ProfileName string `json:",omitempty"`
 
+	// AutoUpdate settings for the node agent.
```

Comment:
complete sentence. there's no verb here.

and what does nil mean?

Reply:
Updated the comment. Nil means auto-updates are disabled (until we're ready to make this opt-out).